cluster/examples/kubernetes/ceph/operator-openshift.yaml

```
@@ -199,6 +201,52 @@ data:
   #     key: node-role.kubernetes.io/etcd
   #     operator: Exists
 
+  # (Optional) Ceph CSI RBD provisioner NodeAffinity.
+  # CSI_RBD_PROVISIONER_NODE_AFFINITY: "role=storage-node; storage=rook, ceph"
+  # (Optional) Ceph CSI RBD provisioner tolerations list. Put here list of taints you want to tolerate in YAML format.
+  # CSI provisioner would be best to start on the same nodes as other ceph daemons.
+  # CSI_RBD_PROVISIONER_TOLERATIONS: |
+  #   - effect: NoSchedule
+  #     key: node-role.kubernetes.io/controlplane
+  #     operator: Exists
+  #   - effect: NoExecute
+  #     key: node-role.kubernetes.io/etcd
+  #     operator: Exists
+  # (Optional) Ceph CSI RBD plugin NodeAffinity.
+  # CSI_RBD_PLUGIN_NODE_AFFINITY: "role=storage-node; storage=rook, ceph"
+  # (Optional) Ceph CSI RBD plugin tolerations list. Put here list of taints you want to tolerate in YAML format.
+  # CSI plugins need to be started on all the nodes where the clients need to mount the storage.
+  # CSI_RBD_PLUGIN_TOLERATIONS: |
+  #   - effect: NoSchedule
```

Let's use a more specific example that shows a way to use these specific affinities and tolerations. Instead of using the same examples everywhere, how about labels such as this? Similar examples could apply in other places.
```suggestion
  #   - key: node.rook.io/rbd
  #   - operator: Exists
```

---

How about if we remove the other two generic tolerations from the examples? A single example seems sufficient.